website-backend/docker/docker_build_ci.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euxo pipefail
+
+SCRIPT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/$(basename "${BASH_SOURCE[0]}")"
+SCRIPT_DIR="$(dirname "$SCRIPT_PATH")"
+
+source "$SCRIPT_DIR/env.sh"
+
+DOCKER_BUILDKIT=1 docker build -f docker/Dockerfile -t "${AWS_ACCOUNT_ID}".dkr.ecr."${AWS_REGION}".amazonaws.com/"${ORG_NAME}"."${IMAGE_NAME}"."${CURRENT_DATE}":git-"${GITHUB_SHA}"
```

Comment:
While there is an `if` in `website-backend/docker/docker_build.sh` to check which platform, it is missing in here. Is it important?

Reply:
I don't think so. 
This file is for CI and we'll be running on linux platforms with x86 as default. 
If there is not a particular reason for doing so I think we can assume the arch and platform by default.